### DISCUSSION TITLE
OOI - Morphia 2.3.0-SNAPSHOT OK with MongoDB 6.0.1 & JARs for 4.8.0

### DISCUSSION THREAD

**Discussion by NOTiFYcouk** [MorphiaOrg/morphia#2164] [General]
OOI I have upgraded my Application to MongoDB 6.0.1 & the JARs for:

On Dec 06, 2022:
```
implementation("org.mongodb:mongodb-driver-core:4.8.2")
implementation("org.mongodb:mongodb-driver-sync:4.8.2")
implementation("org.mongodb:bson:4.8.2")

```
All works fine with Morphia 2.3.0-SNAPSHOT on Openjdk 19.0.1 & WildFly 27.0.1.Final.

20/12/2022: All works fine with:

`implementation 'dev.morphia.morphia:morphia-core:2.3.0'`

**Comment by evanchooly** [comment]
I don't understand the question/problem here...

**Reply by NOTiFYcouk** [reply]
There wasn't - just an "out of interest" comment. Maybe lost in translation ... 

Was looking at:
"Morphia should work fine against 5.x and even 6.x. You can see the test matrix [here](https://github.com/MorphiaOrg/morphia/actions/runs/3109133776)."

**Reply by evanchooly** [reply]
OK, good.  I was afraid I was missing something but that's good to know.
